cpp/mg_utility/mg_graph.hpp

```diff
@@ -241,23 +241,27 @@ class Graph : public GraphView<TSize> {
   ///
   /// @param node_id Memgraphs's inner ID
   ///
-  TSize GetInnerNodeId(std::uint64_t memgraph_id) {
+  TSize GetInnerNodeId(std::uint64_t memgraph_id) const override {
     if (memgraph_to_inner_id_.find(memgraph_id) == memgraph_to_inner_id_.end()) {
       throw mg_exception::InvalidIDException();
     }
-    return memgraph_to_inner_id_[memgraph_id];
+    return memgraph_to_inner_id_.at(memgraph_id);
   }
 
   ///
   /// Returns the Memgraph database ID from graph view
   ///
   /// @param node_id view's inner ID
   ///
-  std::uint64_t GetMemgraphNodeId(TSize node_id) {
+  std::uint64_t GetMemgraphNodeId(TSize node_id) const override {
     if (inner_to_memgraph_id_.find(node_id) == inner_to_memgraph_id_.end()) {
       throw mg_exception::InvalidIDException();
     }
-    return inner_to_memgraph_id_[node_id];
+    return inner_to_memgraph_id_.at(node_id);
+  }
+
+  bool NodeExists(TSize memgraph_id) const override {
+    return memgraph_to_inner_id_.find(memgraph_id) != memgraph_to_inner_id_.end();
```

Comment:
```suggestion
  bool NodeExists(std::uint64_t memgraph_id) const override {
    return memgraph_to_inner_id_.find(memgraph_id) != memgraph_to_inner_id_.end();
```
As this function expects memgraph ids, I think its parameter should be `std::uint64_t` instead of `TSize`.

Reply:
Correct, this is a nice catch!